trappy/ftrace.py

```diff
@@ -88,6 +145,13 @@ def __init__(self, name="", normalize_time=True, scope="all",
         if normalize_time:
             self.normalize_time()
 
+        if self.__class__.disable_cache: return
+
+        self._create_trace_cache()
+        for trace_class in self.trace_classes:
+            csv_file = self._get_csv_path(trace_class)
+            trace_class.write_csv(csv_file)
+
```

Comment:
Ok, I'll catch OSError and print a warning in this case instead of just failing.